src/expression/parse.js

```diff
@@ -916,17 +960,19 @@ export const createParse = /* #__PURE__ */ factory(name, dependencies, ({
 
     while (hasOwnProperty(operators, state.token)) {
       name = state.token
+      const source = tokenSource(state)
       fn = operators[name]
 
       getTokenSkipNewline(state)
 
       if (name === 'in' && state.token === '') {
         // end of expression -> this is the unit 'in' ('inch')
-        node = new OperatorNode('*', 'multiply', [node, new SymbolNode('in')], true)
+        // no source mapping because this * operator is not explicitly in the source expression
+        node = new OperatorNode('*', 'multiply', [node, withSources(new SymbolNode('in'), [source])], true)
```

Comment:
Thinking aloud here: would it make sense to add a source with empty `text` here?

Reply:
I think it depends on whether we can come up with a meaningful `index` value. For other implicit nodes, there's a clear place where the symbol *would* be, EG `(2)(3)` has an implicit multiplication "between" `(2)` and `(3)`

In the case above, it seemed more like this node existed as a way to get the interpreter to perform the correct behavior and not because it has a place in the original expression

That being said, I'm fine with setting the index to something arbitrary like `index: -1` if only for the sake of ensuring that every parsed node has a source

Reply:
Hm, you're right, this is just an odd case creating two nodes from a single operator. Let's leave it as you've implemented it alreay, with the source attached to the inner `SymbolNode` and the outer `OperatorNode` not having a source.